lcd_test/helpers.go

```diff
@@ -151,8 +151,7 @@ func defaultGenesis(config *tmcfg.Config, nValidators int, initAddrs []sdk.AccAd
 
 	// append any additional (non-proposing) validators
 	var genTxs []auth.StdTx
-	var genAccounts []authexported.GenesisAccount
-
+	genAccounts := make([]authexported.GenesisAccount, 0)
```

Comment:
don't preallocate to 0, just add nolint

Reply:
even better, you already know the size to allocate: `len(initAddrs) + nValidators`

Reply:
`len(initAddrs) + nValidators` this seems to be causing most the tests in lcd_test to fail

Reply:
are you setting values in the array correctly? i.e. `genAccounts[genAccsIdx] = genAccount`

Reply:
^^ @marbar3778